bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
These lines prevented me from emitting an event in a callback. I have looked up the aura.js repo, and these lines are gone there also, and `this.stopped` doesn't seem to be used in this file elsewhere, so I guess that should be fine.